schema.lua

```diff
@@ -17,10 +18,19 @@ local function claim_check(value, conf)
   end
 end
 
+local function check_user(anonymous)
```

Comment:
Again, based on the official `jwt` plugin. [See here](https://github.com/Kong/kong/blob/master/kong/plugins/jwt/schema.lua#L3)